### ISSUE TITLE
Implement Font Variation

### ISSUE THREAD

**Issue by KrasnayaPloshchad** [georgd/EB-Garamond#173]
With this implementation it would be useful to make more more optical sizes come true.

**Comment by KrasnayaPloshchad** [comment]
See: https://learn.microsoft.com/en-us/typography/opentype/spec/otvaroverview

**Comment by georgd** [comment]
Yes, that's the plan after merging the two development branches.
